src/main/java/com/poliscrypts/api/utility/ContactHelper.java

```diff
@@ -0,0 +1,35 @@
+package com.poliscrypts.api.utility;
+
+import com.poliscrypts.api.entity.Contact;
+import com.poliscrypts.api.entity.ContactType;
+import com.poliscrypts.api.enumeration.ContactTypeEnum;
+import com.poliscrypts.api.exception.ContactException;
+import com.poliscrypts.api.repository.ContactTypeRepository;
+import org.springframework.context.MessageSource;
+
+import java.util.Locale;
+
+public class ContactHelper {
```

Comment:
instead of make such a helper and a contact type, i think that a contact that have a TVA number is a freelancer, otherwise he is an employee :)
KEEP IT SIMPLE!